xml/selinux.xml

```diff
@@ -417,17 +341,83 @@ system_u:object_r:var_t var</screen>
     </imageobject>
    </mediaobject>
   </figure>
-
+  
   <para>
-   After installing the &selnx; packages, you need to modify the &grub; boot
-   loader. Do this from &yast;, select <menuchoice>
-   <guimenu>System</guimenu> <guimenu>Boot Loader</guimenu> <guimenu>Kernel
-   Parameters</guimenu></menuchoice>. Now add the following parameters to
-   the <guimenu>Optional Kernel Command Line Parameters</guimenu>:
+    This does not install a reference policy. See 
+    <xref linkend="sec-selinux-getpolicy"/> for information on
+    installing an &opensuse; reference policy for testing. 
   </para>
+</sect1>
 
-<screen>security=selinux selinux=1 enforcing=0</screen>
-
+  <sect1 xml:id="sec-selinux-getpolicy">
+  <title>Installing an &selnx; reference policy</title>
+  <para>
+   The policy is an essential component of &selnx;. &productname; &productnumber;
+   does not include a default or reference policy, and you must build a 
+   policy that is customized for your installation. &selnx; policies
+   should be customized for your particular needs; consult your &suse;
+   support engineer for assistance.
+  </para>
+  <para>
+    One way to get a reference policy to install and test is to download
+    it from 
+    <link xlink:href="https://download.opensuse.org/repositories/security:/SELinux/"/>. This provides repositories for &slea; and &opensuse;. Copy
+    the repository link that matches your &slea; version, and add it with
+    zypper:
+  </para>
+  <screen>&prompt.sudo;<command>zypper ar -f https://download.opensuse.org/repositories/security:/SELinux/SLE_15_SP&product-sp;/ Security-SELinux</command>
+      </screen>
+  <para>
+    If you prefer &yast;, use the Software Repositories module.
+  </para>
+  <para>
+    This repository provides a number of additional packages, including
+    reference policies, such as:
+  </para>
+  <screen>
+selinux-autorelabel
+selinux-policy 
+selinux-policy-devel
+selinux-policy-doc
+selinux-policy-minimum 
+selinux-policy-mls
+selinux-policy-sandbox
+selinux-policy-targeted
+setools-gui
+setroubleshoot
+setroubleshoot-plugins
+setroubleshoot-server
+</screen>
+  <para>
+   Install all of the additional packages. 
```

Comment:
Does "all" also mean all policies?

It would be good to know what the policies do and what the additional packages are for. At least a hint (maybe the text provided in the package descriptions)

Reply:
just recommend installing selinux-policy-targeted, selinux-policy-devel and selinux-autorelabel, not the rest

Reply:
:+1: